libraries/AP_HAL_ChibiOS/hwdef/Aocoda-RC-H743Dual/hwdef.dat

```diff
@@ -0,0 +1,218 @@
+PA9  USART1_TX USART1
+PA10 TIM1_CH3 TIM1 RCININT PULLDOWN LOW
+
+# as an alternative config setup the RX1 pin as a uart. This allows
+# for bi-directional UART based receiver protocols such as FPort
+# without any extra hardware
+PA10 USART1_RX USART1 ALT(1)
+
+# USART2 (GPS), SERIAL2
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+
+# USART3 (VTX), SERIAL3
+PD8 USART3_TX USART3
+PD9 USART3_RX USART3
+
+# UART4 (Telemetry), SERIAL4
+PB8 UART4_RX UART4
+PB9 UART4_TX UART4
+
+# USART6 (ESC Telemetry), SERIAL6
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+
+# UART7 (spare), SERIA7
+PE7 UART7_RX UART7
+PE8 UART7_TX UART7
+
+# UART8 (Optical Flow), SERIAL8
+PE0 UART8_RX UART8
+PE1 UART8_TX UART8
+
+# Motors
+PB0  TIM3_CH3 TIM3  PWM(1)  GPIO(50) BIDIR   //S1
+PB1  TIM3_CH4 TIM3  PWM(2)  GPIO(51)         //S2
+PA0  TIM2_CH1 TIM2  PWM(3)  GPIO(52) BIDIR   //S3
+PA1  TIM2_CH2 TIM2  PWM(4)  GPIO(53)         //S4
+PA2  TIM5_CH3 TIM5  PWM(5)  GPIO(54) BIDIR   //S5
+PA3  TIM5_CH4 TIM5  PWM(6)  GPIO(55)         //S6
+PD12 TIM4_CH1 TIM4  PWM(7)  GPIO(56) BIDIR   //S7
+PD13 TIM4_CH2 TIM4  PWM(8)  GPIO(57)         //S8
+PD14 TIM4_CH3  TIM4  PWM(9)  GPIO(58) NODMA  //S9
+PD15 TIM4_CH4  TIM4  PWM(10) GPIO(59) NODMA  //S10
+PE5  TIM15_CH1 TIM15 PWM(11) GPIO(60) NODMA  //SE1
+PE6  TIM15_CH2 TIM15 PWM(12) GPIO(61) NODMA  //SE2
+#PA8  TIM1_CH1 TIM1  PWM(13) GPIO(62) # for WS2812 LED  this will be affects RC input
+
+# PWM output for buzzer/Beeper
+PA15 BUZZER OUTPUT LOW PULLDOWN GPIO(80)
+define HAL_BUZZER_PIN 80
+
+# GPIOs for 9V DCDC control (HIGH:on; LOW:off)
+PD10  PINIO1 OUTPUT GPIO(81) HIGH
+
+# GPIOs for pinio // not used
+#PD11  PINIO2 OUTPUT GPIO(82) LOW
+
+# DMA Priority
+DMA_PRIORITY S* TIM3* TIM2*
+DMA_NOSHARE SPI1* SPI4* TIM3* TIM2*
+
+# use last 2 pages for flash storage
+# H743 has 16 pages of 128k each
+STORAGE_FLASH_PAGE 14
+define HAL_STORAGE_SIZE 32768
+
+# spi devices
+SPIDEV mpu6000       SPI1 DEVID1 IMU1_CS      MODE3   1*MHZ   4*MHZ
+SPIDEV bmi270_1      SPI1 DEVID1 IMU1_CS      MODE3  10*MHZ  10*MHZ
+SPIDEV bmi270_2      SPI4 DEVID1 IMU2_CS      MODE3  10*MHZ  10*MHZ
+SPIDEV dataflash     SPI3 DEVID1 FLASH_CS     MODE3 104*MHZ 104*MHZ
+SPIDEV osd           SPI2 DEVID4 MAX7456_CS   MODE0  10*MHZ  10*MHZ
+
+# two BMI270 or one MPU6000 IMU
+IMU Invensense SPI:mpu6000 ROTATION_YAW_270
+IMU BMI270 SPI:bmi270_1 ROTATION_ROLL_180_YAW_90
+IMU BMI270 SPI:bmi270_2 ROTATION_PITCH_180
+
+# DPS310 integrated on I2C2 bus, multiple possible choices for external barometer
+BARO MS56XX I2C:0:0x77
+BARO DPS310 I2C:0:0x76
+BARO BMP280 I2C:0:0x76
+
+# no built-in compass, but probe the i2c bus for all possible
+# external compass types
+define ALLOW_ARM_NO_COMPASS
+define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+define HAL_I2C_INTERNAL_MASK 0
+define HAL_COMPASS_AUTO_ROT_DEFAULT 2
+
+# setup for OSD
+define OSD_ENABLED 1
+define HAL_OSD_TYPE_DEFAULT 1
+ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
+
+# support VTX IRC tramp protocol
+define AP_TRAMP_ENABLED 1

```

Comment:
this is enabled by default on 2MB

Reply:
OK, I'll remove the tramp. Thanks for the tip.